src/SkyApm.Agent.Hosting/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -40,11 +40,13 @@
 using SkyApm.PeerFormatters.SqlClient;
 using SkyApm.PeerFormatters.MySqlConnector;
 using ILoggerFactory = SkyApm.Logging.ILoggerFactory;
+using System.IO;
 
 namespace Microsoft.Extensions.DependencyInjection
 {
     public static class ServiceCollectionExtensions
     {
+        private static string oldConfigName = "skywalking.json";

```

Comment:
When did we use this name? From which version?

Reply:
skywalking.json？？？

Reply:
Yes. 2.1 to 2.2?

Reply:
It used to be this sywalk.json, I used the old version to upgrade to 2.2, but apache-skywalk-APM-9.5.0 view is empty, but I have dropped to 2.1, view the source code only to find that the configuration name has changed